packages/amplify-graphql-api-construct/src/internal/construct-exports.ts

```diff
@@ -33,7 +41,7 @@ export const getGeneratedResources = (scope: Construct): AmplifyGraphqlApiResour
   const cfnResolvers: Record<string, CfnResolver> = {};
   const cfnFunctionConfigurations: Record<string, CfnFunctionConfiguration> = {};
   const cfnDataSources: Record<string, CfnDataSource> = {};
-  const tables: Record<string, Table> = {};
+  const tables: Record<string, ITable> = {};
```

Comment:
This does not change the final output type for tables, which is already defined as `ITable` [here ](https://github.com/aws-amplify/amplify-category-api/blob/cd23fb9a9ad982198c10ebfba07d285871e5d62a/packages/amplify-graphql-api-construct/src/types.ts#L747-L750)